### ISSUE TITLE
Accommodate certificate name in KeyLocator in /localhop/nfd/rib validation rules

### ISSUE THREAD

**Issue by yoursunny** [WU-ARL/NDN_Ansible#35]
Currently, command validation rules for `/localhop/nfd/rib` command are written as:

```
rule
{
    for interest
    filter
    {
        type name
        regex ^[<localhop><localhost>]<nfd><rib>[<register><unregister>]<><><>$
    }
    checker
    {
        type customized
        sig-type rsa-sha256
        key-locator
        {
            type name
            regex ^<>*<KEY><>$
        }
    }
}
```

This means, the validator can only accept a signed Interest if its KeyLocator contains key name, but would reject a signed Interest if its KeyLocator contains certificate name.

Since [ndn-cxx Feature 5112](https://redmine.named-data.net/issues/5112), KeyLocator would contain certificate name. Additionally, legacy client may continue to send KeyLocator with key name.
Thus, this rule should be relaxed to accept either key name or certificate name as KeyLocator.

[NFD Feature 5114](https://redmine.named-data.net/issues/5114) will provide specific examples on what to write in `nfd.conf`. After that issue is closed, `nfd.conf.j2` in this codebase should be updated accordingly and deployed to testbed.
It is not necessary to wait for NFD release, and this change would not affect previous versions of end hosts.

**Comment by yoursunny** [comment]
I believe the `nfd.conf` changes are ready for a test deployment.

Please see the sample configuration in:

* https://redmine.named-data.net/issues/5114#note-6
* [Gerrit Change 6331](https://gerrit.named-data.net/gitweb?p=NFD.git;a=blob;f=nfd.conf.sample.in;h=6a84a89219fbb7d22cafa5d881ad43cf47d71ed6;hb=0b4bc8a5ba3733746193e2a6057818ea68c39051#l356)

**Comment by yoursunny** [comment]
Implemented in b8d1f0487d1a55475b97e1a9ed57aba906d39ff3
